src/Link/index.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2020 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import React from 'react'
+import '@spectrum-css/link'
+import classNames from 'classnames'
+
+export const Link = ({ children, className, href, ...props }) => (
+  <a {...props} href={href} className={classNames(className, 'spectrum-Link')}>
+    {children}
+  </a>
+)
```

Comment:
In order to make the Link component more generic we could wrap the Gatsby Link component. The benefit here is it will work in development and production mode. In production the site prefix path is added dynamically. Something like:

```javascript
import React from 'react'
import '@spectrum-css/link'
import classNames from 'classnames'
import { Link as GatsbyLink } from 'gatsby'

export const Link = ({ children, className, href, ...props }) => (
    <GatsbyLink
        to={href}
        className={classNames(className, 'spectrum-Link')}
        {...props}
      >
    {children}
  </GatsbyLink>
)

```

Reply:
Link is only for local not external links so we would need to handle that logic as well here. I prefer not to and instead use plugins like remark-external-links and gatsby-plugin-catch-links to handle it "magically".